setup.cfg

```diff
@@ -2,36 +2,13 @@
 [tool:pytest]
 # Function starting with the following pattern are considered for test cases.
 python_functions=test_
-
-# pytest pycodestyle configuration
-codestyle_max_line_length = 88
-codestyle_ignore = E302 W503 E203 E501 E265 E402 E251 E211
-codestyle_exclude =
-    rasa/core/policies/tf_utils.py
-    rasa/core/policies/__init__.py
```

Comment:
I think is this legacy config, we need to review if it breaks something in the CI